insta-clone/src/App.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data'
+import SearchBarContainer from './components/SearchBar/SearchBarContainer';
+import PostContainer from './components/PostContainer/PostContainer';
+
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {
+      newArray: [],
+      searchedInput: "",
+      copyArray: []
+    }
+  }
+
+  componentDidMount() {
+    this.setState({
+      newArray:dummyData,
```

Comment:
I suggest using a more specific name for your newArray key, such as userData, as newArray is a very generic